templates/flavors/windows/machine-deployment-windows.yaml

```diff
@@ -27,12 +27,12 @@ kind: AzureMachineTemplate
 metadata:
   name: "${CLUSTER_NAME}-md-win"
   annotations:
-    runtime: dockershim
+    runtime: containerd
```

Comment:
This template file is very specific to the dockershim implementation.  The existing ones in here should be deleted and the files at https://github.com/kubernetes-sigs/cluster-api-provider-azure/tree/main/templates/flavors/windows-containerd should be renamed to windows to do the replacement properly.

Reply:
done